models/fates/inst/create_CLM5-FATES_1x1br_refrun.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+#
+#
+#=============================================================================================
+#
+# Generate CLM5-FATES reference run for PEcAn site-scale simulations
+# 
+# Users should update all of the lines with -USER UPDATE to match their local environment
+#
+# --- Last updated by Shawn P. Serbin <sserbin@bnl.gov>
+#=============================================================================================
+
+# Define CTSM-FATES options
+export MODEL_SOURCE=/data/software/CTSM/ctsm                   	# CTSM/CLM/ELM source directory - USER UPDATE
+export PROJECT=PEcAn_simulations                               	# project or run tag
+export COMPSET=I2000Clm50FatesGs				# define compset for run
+export RES=1x1_brazil						# set grid resolution
+export MAC=modex                                             	# Name your machine - USER UPDATE
+export COMPILER=gnu                                          	# Name your compiler - USER UPDATE
+export CASEROOT=/data/software/FATES/FATES_refrun               # Where the build is generated. The ref case location - USER UPDATE
+
+# Set CLM & FATES parameter path and param files here:
+export FATES_PARAM_FILE_PATH=/data/software/CTSM/fates_parameter_files 			# - USER UPDATE
+export FATES_PARAM_FILE=fates_params_2troppftclones.c171018_sps.nc			# - USER UPDATE
+export CLM_PARAM_FILE_PATH=/data/Model_Data/cesm_input_datasets/lnd/clm2/paramdata	# - USER UPDATE
+export CLM_PARAM_FILE=clm5_params.c171117.nc						# - USER UPDATE
+
+# Define output options
+export date_var=$(date +%s)
+echo "*** Build start: ${date_var} "
+export TAG='PEcAn_ref_run03'
+export MODEL_VERSION=CLM5-FATES
+export CLM_HASH=`(cd ${MODEL_SOURCE};git log -n 1 --pretty=%h)`
+export FATES_HASH=`(cd ${MODEL_SOURCE}/src/fates;git log -n 1 --pretty=%h)`
+export GIT_HASH=C${CLM_HASH}-F${FATES_HASH}
+export CASE_NAME=${CASEROOT}/${MODEL_VERSION}_${TAG}
```

Comment:
Unless you need these variables outside of this script (i.e. in the parent process), you shouldn't need `export`, and in fact should probably avoid it to not pollute/confuse the environment.

I would try dropping `export` and seeing if the script still works.

Reply:
My understanding is that if you run this script followed by another script that does depend on these settings, then you need to keep `export`. I.e. If the workflow is something like:

```
./create_CLM5-FATES...
./run_model.sh   # This script depends on CLM_HASH, FATES_HASH, etc...
```

However, if these variables are only used _within_ the `create_CLM45...` script, then you should remove the export.